tracpro/msgs/views.py

```diff
@@ -170,8 +169,8 @@ def post(self, request, *args, **kwargs):
                 # Wait 1 second to allow message to reach server then run the task
```

Comment:
No I left the comment by mistake. We no longer need the sleep at least from my functional testing. I will remove it.